.github/workflows/deploy.yaml

```diff
@@ -16,9 +16,9 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - name: Setup api
-        uses: actions/setup-node@v3
+        uses: actions/setup-node@v4
         with:
-          node-version: '18'
+          node-version: '20'
```

Comment:
Should also upgrade the lambda version in the CDK stacks?

Reply:
Updated :)